docs/zenith/developer/python/419481-quickstart.md

```diff
@@ -73,19 +72,9 @@ pip install -e ./sdk
 ```
 
 :::tip
-As always, you can use any virtual environment manager, including for example
-[Poetry](https://python-poetry.org). Just make sure you add `./sdk` as a
-**development dependency**.
+As always, you can use any virtual environment manager, including [Poetry](https://python-poetry.org) and others. Just make sure you add `./sdk` as a development dependency. If you place the virtual environment inside the module, don't forget to add it to `.gitignore` and to `"exclude"` in `dagger.json`.
```

Comment:
I emphasized **development dependency** because it's important but wasn't sure if it needs an explanation in this guide. 

The runtime container will install the module with `pip install .`, but it will already have installed the codegen'ed SDK at that point already. We only need to install `./sdk` locally then but must put it in the non-primary group so it doesn't get picked up by the runtime.

For example, in Poetry you'd add it to the `dev` group like so:

```toml
[tool.poetry.dependencies]
python = "^3.11"

[tool.poetry.group.dev.dependencies]
dagger-io = {path="./sdk", develop=true}
```

This way it's installed locally when you do `poetry install`, but not in the runtime.

Reference: https://github.com/helderco/daggerverse/blob/979ee1062037f465e07257a4b740f8a3adf54f0d/poetry/pyproject.toml#L8-L12

Reply:
Seems important to me but I'm not sure the quickstart is the best place for it. Would you like to add it to the miscellaneous section of the advanced guide?

Reply:
Yeah, that's what I thought. I think it needs a section somewhere on dependencies in general. There's more than that, like for example the choice of project name, as you can see in https://github.com/helderco/daggerverse/tree/main/poetry. We can do it in a follow up though.